setup.md

```diff
@@ -40,10 +40,10 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
 - [Docker](https://docs.docker.com/get-docker/)
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [Git](https://git-scm.com/downloads)
-- [Go](https://go.dev/doc/install)
+- [Go](https://go.dev/doc/install) (v1.21)
```

Comment:
isn't the latest version currently v1.20.6? We could add "v1.20+"

Reply:
When I ran it with v1.13 it failed with a variety of errors. 
Updating to the latest release told me that I needed v1.20.1 
Sorry for the typo. Shall I correct and issue another pull request?

Reply:
You can just update this pull request with the correct one. I am not sure why the DCO fails for you. Might need to double check your git config